README.md

```diff
@@ -1,20 +1,21 @@
-# Go driver for [Babelfish](https://github.com/bblfsh/bblfshd) ![Driver Status](https://img.shields.io/badge/status-beta-dbd25c.svg) [![Build Status](https://travis-ci.org/bblfsh/go-driver.svg?branch=master)](https://travis-ci.org/bblfsh/go-driver) ![Native Version](https://img.shields.io/badge/go%20version-1.10-aa93ea.svg) ![Go Version](https://img.shields.io/badge/go%20version-1.10-63afbf.svg)
+# Go driver for [Babelfish](https://github.com/bblfsh/bblfshd) ![Driver Status](https://img.shields.io/badge/status-beta-dbd25c.svg) [![Build Status](https://travis-ci.org/bblfsh/go-driver.svg?branch=master)](https://travis-ci.org/bblfsh/go-driver) ![Native Version](https://img.shields.io/badge/go%20version-1.10-aa93ea.svg) ![Go Version](https://img.shields.io/badge/go%20version-1.12-63afbf.svg)
 
 Development Environment
 -----------------------
 
 Requirements:
 - `docker`
-- Go 1.11+
-- SDK dependencies _(dep ensure --vendor-only)_
+- Go 1.12+
```

Comment:
Is this not affected by our two-version compatibility promise because it's the developer environment rather than the release?

Reply:
Two version compatibility is only for libraries, while the driver is considered an application, I guess.